libr/util/format.c

```diff
@@ -57,17 +57,30 @@ static float updateAddr(const ut8 *buf, int len, int endian, ut64 *addr, ut64 *a
 
 static int r_get_size(RNum *num, ut8 *buf, int endian, const char *s) {
 	int size = 0, len = strlen (s);
-	ut64 addr;
-
 	if (s[0] == '*' && len >= 4) { // value pointed by the address
+		ut64 addr;
 		int offset = (int)r_num_math (num, s + 1);
 		(void)updateAddr (buf + offset, 999, endian, &addr, NULL);
 		return addr;
+	}
+	// flag handling doesnt seems to work here
+	return size = r_num_math (num, s);
+}
+
+static void r_print_format_u128(const RPrint* p, int endian, int mode,
```

Comment:
by looking at the other `r_print_format_*` functions it seems something is missing from this. Could you check? I think you don't support json and some other stuff.